src/services/package-version.js

```diff
@@ -0,0 +1,7 @@
+const packageVersion = async (name) => {
+    const request = await fetch(`http://registry.npmjs.com/-/v1/search?text=${name}&size=20`)
```

Comment:
Should be a try/catch (or similar) here, in case this api becomes offline

Reply:
Hi Jhow! I've included a catch startment for this case.